jest.config.json

```diff
@@ -10,7 +10,7 @@
     "\\.svg\\?(component|source)": "<rootDir>/frontend/test/__mocks__/svgMock.jsx"
   },
   "transformIgnorePatterns": [
-    "<rootDir>/node_modules/(?!rehype-external-links/)"
+    "<rootDir>/node_modules/(?!(rehype-external-links|hast.*|devlop|property-information|comma-separated-tokens|space-separated-tokens|vfile|vfile-message|html-void-elements|stringify-entities|character-entities-html4)/)"
```

Comment:
The hast util libraries and many of their peerDependencies (most of which are written by the same author) are uncompiled and use ESModules `import` and `export` syntax.

This syntax is not compatible with Jest (unless we enable [experimental support](https://jestjs.io/docs/ecmascript-modules)). Jest instead transforms modules itself, but by default will ignore all of `node_modules`, since it would have a negative impact on performance.

Like we did previously for `rehype-external-links`, we add the hast libraries and their dependencies to the regex pattern, to tell Jest to transform these libraries, so it doesn't error upon encountering ESModules syntax.